hoomd/md/pytest/test_lj_equation_of_state.py

```diff
@@ -0,0 +1,127 @@
+import hoomd
+import pytest
+import hoomd.conftest
+import numpy
+import math
+
+# Selected state points in the high density fluid. Both of these state points
+# are in single phase regions of the phase diagram and suitable for NVT/NPT and
```

Comment:
Number of statepoints seems inaccurate currently?

Reply:
I made it `point(s)` so we don't have to change the comment if we decide to add more.